.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,24 @@
+name: Node CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x]
```

Comment:
@mbostock if you want to rebase #205 off of this branch and add `, 12.x` to here, you’ll get node v12 test runs.

Reply:
Done.